File: clj/eval.clj
; Steve Phillips / elimisteve
; 2011.01.07

(eval
 `(+ 10
	 ~(* 3 2)))
; 16

(eval
 '(+ 10
	 (* 3 2)))
; 16

(eval
 (let [x '(+ 2 3)]
   `(~@x 1)))
; 6

(= `~'+ '+)
; true

;(defm )

File: go/mostlybroken/eval.go
// Steve Phillips / elimisteve
// 2011.06.08

package main

import (
    "exp/eval"
    "fmt"
)

func main() {
    world := eval.NewWorld()
    code, err := world.Compile(nil, `md5.New()`)
    if err == nil{
        fmt.Printf("code == %v\n", code)
    } else{
        fmt.Printf("err == %v\n", err)
    }
    
}